inc/saivlan.h

```diff
@@ -281,6 +281,18 @@ typedef enum _sai_vlan_attr_t
     /** Custom range base value */
     SAI_VLAN_ATTR_CUSTOM_RANGE_START = 0x10000000,
 
+    /**
+     * @brief IGMP Snooping enable or disable control for VLAN
+     *
+     * IGMP Snooping enable control for VLAN. Default is
+     * disabled
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
+     */
+    SAI_VLAN_ATTR_CUSTOM_IGMP_SNOOPING_ENABLE,
```

Comment:
Hi @lguohan , I tried the changes you suggested. Now I am getting the following error
 ASSERT FAILED (on line 178): expected true 'value < 0x10000': enum value is too big, range?
Makefile:52: recipe for target 'all' failed
Should the SAI_VLAN_ATTR_CUSTOM_RANGE_START value be changed < 0x10000 ?